src/components/interfaces/MOBILE_API.xml

```diff
@@ -2014,6 +2014,13 @@
                 <param name="speed" type="Float" minvalue="0" maxvalue="500" mandatory="true" since="2.0" until="5.0"/>
             </history>
         </param>
+        <param name="shifted" type="Boolean" mandatory="false" since="5.0">
```

Comment:
@LuxoftAKutsan I believe we currently have a PR against develop which will change the version from `5.1` to `5.2`(https://github.com/smartdevicelink/sdl_core/pull/2927). Also the new `shifted` parameter will have to have `since=5.2` which might cause problems if the RAPI version is not incremented.